cfgov/data_research/models.py

```diff
@@ -15,11 +15,43 @@
 logger = logging.getLogger(__name__)
 
 
+class ConferenceRegistrationQuerySet(models.QuerySet):
+    # TODO: In Django 1.9+, this logic can be optimized through use of
+    # Django's built-in Postgres JSONFields. This will require migrating
+    # the details field of the ConferenceRegistration model.
+    #
+    # return self.filter(
+    #     details__attendee_type=self.IN_PERSON
+    # )
+    def in_person(self):
+        return filter(
+            lambda a: (
+                a.details.get('attendee_type') ==
+                ConferenceRegistration.IN_PERSON
+            ),
+            self
+        )
+
+    def virtual(self):
+        return filter(
+            lambda a: (
+                a.details.get('attendee_type') ==
+                ConferenceRegistration.VIRTUAL
+            ),
+            self
+        )
+
+
 class ConferenceRegistration(models.Model):
+    IN_PERSON = 'In person'
```

Comment:
To avoid duplication, can we remove [these hardcoded values](https://github.com/cfpb/cfgov-refresh/blob/master/cfgov/data_research/forms.py#L36) on the registration form class and instead use these new ones on the model?

Reply:
Gah, yeah. I meant to do that.